src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/230045957-e3759431-0fa4-4772-8e6c-893d6951b284.png)
When calling this method, we should not receive a string, but immediately print information in the console.